include/odp/api/spec/crypto.h

```diff
@@ -698,6 +755,14 @@ typedef struct odp_crypto_cipher_capability_t {
 	/** IV length in bytes */
 	uint32_t iv_len;
 
+	/** Cipher is operating in bitstring mode
+	 *
+	 * This cipher works on bit strings, rather than sequences of bytes:
+	 * cipher_range's length parameter in odp_crypto_op_param_t and
+	 * odp_crypto_packet_op_param_t will be expressed in bits, rather than
+	 * bytes */
+	odp_bool_t bitstring_mode;
```

Comment:
I'd call this bit_mode. We don't call byte length/offset defined data as "byte strings", so I would not introduce "bit string" term either. Also, length and offset should be give in the same unit (normally both are bytes, in bit mode both are bits).

Just define that: "in bit mode cipher_range parameters (length and offset) are given in bits, instead of bytes"